org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/OurSyntaxDocument.java

```diff
@@ -39,6 +39,8 @@
  * Graphical syntax-highlighting StyledDocument.
  * <p>
  * <b>Thread Safety:</b> Can be called only by the AWT event thread
+ *
+ * @modified: Nuno Macedo, Eduardo Pessoa // [HASLab] electrum-temporal
```

Comment:
this essentially adds the new language keywords and forbids primes ' in identifiers